library/src/math/style.rs

```diff
@@ -178,6 +178,84 @@ pub fn bb(
         .into()
 }
 
+pub fn sized() -> Module {
+    let mut scope = Scope::new();
+    scope.define("display", display);
+    scope.define("inline", inline);
+    scope.define("script", script);
+    scope.define("scriptscript", scriptscript);
+    Module::new("sized").with_scope(scope)
+}
+
+/// Forced display style in math.
+///
+/// ## Example
+/// ```example
+/// $ sized.display(sum_i (x_i mu_i)/sigma_i^2)/(sum_i (mu_i^2)/sigma_i^2) $
+/// ```
+///
+/// Display: Sized Display
+/// Category: math
+/// Returns: content
+#[func]
+fn display(body: Content) -> Value {
+    MathStyleElem::new(body)
+        .with_size(Some(MathSize::Display))
+        .pack()
+        .into()
+}
+
+/// Forced inline (text) style in math.
+///
+/// ## Example
+/// ```example
+/// $ sized.inline(sum_i (x_i mu_i)/sigma_i^2)/(sum_i (mu_i^2)/sigma_i^2) $
+/// ```
+///
+/// Display: Sized Inline
+/// Category: math
+/// Returns: content
+#[func]
+fn inline(body: Content) -> Value {
+    MathStyleElem::new(body).with_size(Some(MathSize::Text)).pack().into()
+}
+
+/// Forced script style in math.
+///
+/// ## Example
+/// ```example
+/// $ script.text(sum_i (x_i mu_i)/sigma_i^2)/(sum_i (mu_i^2)/sigma_i^2) $
+/// ```
+///
+/// Display: Sized Script
+/// Category: math
+/// Returns: content
+#[func]
+fn script(body: Content) -> Value {
+    MathStyleElem::new(body)
+        .with_size(Some(MathSize::Script))
+        .pack()
+        .into()
+}
+
+/// Forced script-script style in math.
```

Comment:
The name "script-script" isn't (IMHO) that intuitive (this isn't your fault!) - perhaps you could add a brief explanation to the docs that this corresponds to (e.g.) a smaller script size than "script"?